functions/api/audit.js

```diff
@@ -21,26 +22,46 @@ router.get("/", isAdmin, async (request, response, next) => {
     let snapshot = await auditCollection.get()
     snapshot.forEach(audit => {
         var auditData = audit.data()
-        auditData[constants.DATE] = auditData[constants.DATE].toDate() 
+        auditData[constants.DATE] = auditData[constants.DATE].toDate()
+        if(auditData[constants.BEFORE] && auditData[constants.AFTER]) {
+            auditData[constants.BEFORE] = utils.formatDate(auditData[constants.BEFORE])
+            auditData[constants.AFTER] = utils.formatDate(auditData[constants.AFTER])
+        }
         audits.audit.push(auditData);
     })
     audits[constants.TOTAL] = snapshot.size;
     logger.debug('Returning audit log to client.');
     response.status(200).send(audits);
 });
 
+// TODO: DELETE ME
```

Comment:
missed to delete? :P

Reply:
No. Actually lots of audit logs get created. So i kept that method to delete docs in audit collection. Will remove it some time later.